src/main/resources/application.yml

```diff
@@ -8,8 +8,16 @@ spring:
       enabled: true
   jpa:
     hibernate:
-      ddl-auto: "create-drop" # FIXME: Just for early development without migrations
-
+      ddl-auto: "create" # FIXME: Just for early development without migrations
+    open-in-view: false
+    generate-ddl: true
```

Comment:
Do we need this line?  It is not used if Hibernate auto-configuration is active. See https://docs.spring.io/spring-boot/docs/3.2.4/reference/htmlsingle/#data.sql.jpa-and-spring-data.creating-and-dropping